analyses/survival-analysis/survival-analysis_immune.Rmd

```diff
@@ -0,0 +1,324 @@
+
+```{r cox_pdl1_library}
+fit_save_model(combined_df, 
+               "CD274 + library",
+               file.path(results_dir, "cox_terms_pdl1_library.RDS"), 
+               model_type = "multivariate")
+
+```
+
+We observe a _highly significant_ library effect, suggesting strong batch effects in the analysis. **Therefore, we will model using only stranded data for the rest of this notebook to ameliorate this influence.**
+
+### Subset to stranded only
+
+```{r stranded}
+combined_df_stranded <- filter(combined_df, library == "stranded")
+
+# how many samples do we now have?
+nrow(combined_df_stranded)
+```
+
+## Re-analyze with stranded only: PDL-1 expression across entire cohort
+
+> There is no signficant effect of PDL-1 expression. Although this P-value could be viewed as "approaching signficance," the large analysis sample size (N=719), this P-value is actually rather large.
+
+
+```{r cox_pdl1}
+fit_save_model(combined_df_stranded, 
+               "CD274",
+               file.path(results_dir, "cox_terms_pdl1_stranded.RDS"))
+
+```
+
+
+## Analysis: quantiseq cell type fractions across entire cohort
+
+> No cell types are meaningfully signficant.
+
+```{r cox_cell_type}
+
+# Define the terms for a quantiseq model. These are used several times in this notebook
+quantiseq_terms <- "B_cell + Macrophage_M1 + Macrophage_M2 + Monocyte + Myeloid_dendritic_cell + Neutrophil + NK_cell + T_cell_CD4_non_regulatory + T_cell_CD8 + T_cell_regulatory_Tregs" 
+
+
+# saving so P-values can be programmatically grabbed
+fitted <- fit_save_model(combined_df_stranded,
+                         quantiseq_terms,
+                         file.path(results_dir, "cox_terms_quantiseq.RDS"),
+                         model_type = "multivariate")
+fitted
+```
+
+Above, no effects are significant at P<0.01. Only one cell type, `T_cell_CD8`, is marginally significant ($P=`r fitted$p.value[fitted$term == "T_cell_CD8"]`$), but it's confidence interval is *huge* so the exact effect is challenging to discern.
+
+## Analysis: quantiseq cell type fractions and PDL-1  across entire cohort
+
+> No effects are significant
+
+```{r cox_cell_type_cd274}
+fit_save_model(combined_df_stranded,
+               paste0(quantiseq_terms, "+CD274"),
+               file.path(results_dir, "cox_terms_quantiseq.RDS"),
+               model_type = "multivariate")
+```
+
+
+## Analysis: quantiseq cell type fractions and PDL-1 for HGG 
+
+> No effects are significant
+
+```{r cox_hgg_cell_type_cd274}
+hgg_groups <- c("High-grade glioma astrocytoma", "Diffuse midline glioma", "Diffuse intrinsic pontine glioma")
+hgg_df <- combined_df_stranded %>%
+  filter(cancer_group %in% hgg_groups)
+
+fit_save_model(combined_df_stranded,
+               paste0(quantiseq_terms, "+CD274"),
+               file.path(results_dir, "cox_hgg_terms_quantiseq.RDS"),
+               model_type = "multivariate")
+```
+
+
```

Comment:
Ah, I see this now. I was looking for the univariat analysis on `hgg_df` for the term `molecular_subtype` (rather than splitting all HGGs in the later portion of the notebook for subtype).

Reply:
(which is below, so this can be removed)

Reply:
We can remove this analysis altogether and I'll approve

Reply:
I removed